<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflineWidget</class>
 <widget class="QWidget" name="OfflineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_ai_first">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>AI FIRST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_human_first">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>HUMAN FIRST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hotseat">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>HOTSEAT MODE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
